threat_intel/__init__.py

```diff
@@ -284,6 +261,100 @@
 }
 
 
+#### URL scan Endpoint
+
+Calls url/scan VirusTotal API endpoint.
+Submit a url or any other Python enumerable containing the URL addresses:
+
+    >>> urls = ["http://www.google.com", "http://www.yelp.com"]
+    >>> vt.post_url_report(urls)
+
+
+#### Hash report endpoint
+
+Calls file/report VirusTotal API endpoint.
+You can request the file reports passing a list of hashes (md5, sha1 or sha2):
+
+    >>> file_hashes = [
+        "99017f6eebbac24f351415dd410d522d",
+        "88817f6eebbac24f351415dd410d522d"
+    ]
+
+    >>> vt.get_file_reports(file_hashes)
+
+will result in:
+
+{
+  "88817f6eebbac24f351415dd410d522d": {
+    "response_code": 0,
+    "resource": "88817f6eebbac24f351415dd410d522d",
+    "verbose_msg": "The requested resource is not among the finished, queued or pending scans"
+  },
+  "99017f6eebbac24f351415dd410d522d": {
+    "scan_id": "52d3df0ed60c46f336c131bf2ca454f73bafdc4b04dfa2aea80746f5ba9e6d1c-1423261860",
+    "sha1": "4d1740485713a2ab3a4f5822a01f645fe8387f92",
+}
+
+
+#### Hash rescan endpoint
```

Comment:
I think we are following the README choice and writing everything non-capitalized (whats the word for that?). Let me know if you oppose to this.